### ISSUE TITLE
Can’t change background for Settings window 

### ISSUE THREAD

**Issue by Jon-Kinne** [MikeSiLVO/skin.aeon.nox.silvo#226]
Trying to customize settings window as I have all the other windows which open when menu item is pressed. Under System>Select Background ,“None” is chosen, but settings window opens with “circuit board” background.

**Comment by MikeSiLVO** [comment]
That’s the menu item background, not the background you see after opening the window. The quickest solution is to go to skin settings, open the side menu, and click on the following options: 

Background > Fallback: System > Solid color backgrounds

Select black.jpg, if that’s your preferred look.

**Comment by Jon-Kinne** [comment]
Thank you very much! I had found that setting when I set up a prior version of Kodi, but when I went to Omega I forgot where it was, so I appreciate your directionJon...remotely via iPhone”If life were meant to be easy, Michelangelo would have painted the floor.”On Oct 12, 2024, at 4:42 PM, MikeSiLVO ***@***.***> wrote:﻿
That’s the menu item background, not the background you see after opening the window. The quickest solution is to go to skin settings, open the side menu, and click on the following options:
Background > Fallback: System > Solid color backgrounds
Select black.jpg, if that’s your preferred look.
